sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+            ''--TBD--'' as "Partner disability benefits amount",
+            ''--TBD--'' as "Partner Canada Student Loan amount",
+            ''--TBD--'' as "Partner child support amount",
+            -- Partner Info Self-Reported by Student
+            applications.data ->> ''estimatedSpouseIncome'' as "Partner estimated income self-reported",
+            -- Family Info
+            student_assessments.workflow_data -> ''calculatedData'' ->> ''familySize'' as "Family size",
+            jsonb_array_length(applications.data -> ''dependants'') as "Number of dependant children in PSE",
+            ''--TBD--'' as "Number of Dep (0-11 y.o. and disabled 12 y.o. or older)",
+            ''--TBD--'' as "Number of Dep (12 y.o.+ NOT disabled) and Other Dep Relatives",
+            applications.data ->> ''supportnocustodyDependants'' as "Has dependants without sole custody flag",
+            -- Institution Info
+            institution_locations.institution_code as "Institution Code (location code)",
+            ''--TBD--'' as "Alphabetical Institution Listing",
+            ''--TBD--'' as "Institution Location",
+            ''--TBD--'' as "Institution Type",
+            ''--TBD--'' as "Institution Contact List",
+            ''--TBD--'' as "De-designation",
+            -- Program Info
+            education_programs.program_name as "Program name",
+            education_programs.credential_type as "Credential type",
+            education_programs.cip_code as "Classification of Instructional Program (CIP) Code",
+            education_programs.field_of_study_code as "Field of Study Code",
+            education_programs.noc_code as "National Occupation Classification",
+            education_programs.sabc_code as "SABC Program Code",
+            education_programs.institution_program_code as "Institution Program Code",
+            (
+                case
+                    when education_programs.program_intensity = ''Full Time and Part Time'' then ''Y''
+                    else ''N''
+                end
+            ) as "Part-Time eligible (Yes/No)",
+            ''--TBD--'' as "Program Delivery ",
+            education_programs.completion_years as "Program Length",
+            ''Credit or Hours Based'' as "Credit or Hours Based",
+            ''--TBD--'' as "Program Regulatory Body",
+            ''--TBD--'' as "Entrance Requirements",
+            ''--TBD--'' as "Program ESL Content Percentage",
+            ''--TBD--'' as "Program offered jointly/ in partnership (Yes/No)",
+            ''--TBD--'' as "Program Work Integrated Learning (WIL) Component (Yes/No)",
+            ''--TBD--'' as "WIL regulatory approval (Yes/No � as applicable)",
+            ''--TBD--'' as "Program Field Trip / Travel Component (Yes/No)",
+            ''--TBD--'' as "Program international exchange component (Yes/No)",
+            ''--TBD--'' as "Program was pending but then aproved",
+            -- Study Period Offering Info
```

Comment:
As discussed adding the fields for the next story,
![image](https://github.com/bcgov/SIMS/assets/77353155/42e7ff9a-9ee3-47b6-a2b7-d7d7049954de)
![image](https://github.com/bcgov/SIMS/assets/77353155/b2f403f5-09c9-44e7-a077-b15949040f43)
![image](https://github.com/bcgov/SIMS/assets/77353155/2d7f4d54-0bcc-492a-9427-db329f3ca819)
![image](https://github.com/bcgov/SIMS/assets/77353155/0dcf5bcb-b81d-46c0-b06a-5214de4adf88)
![image](https://github.com/bcgov/SIMS/assets/77353155/4f3f81d5-5248-4cb7-b7f4-ea16e60d8766)
![image](https://github.com/bcgov/SIMS/assets/77353155/ac4fa6b8-391f-4a0d-8627-507286ee2a21)
![image](https://github.com/bcgov/SIMS/assets/77353155/6c7279e7-5469-43d2-a06d-cd47603671c2)
![image](https://github.com/bcgov/SIMS/assets/77353155/b4da132d-3472-48ea-9a13-6289a9268d31)